kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DeltaTableWritesSuite.scala

```diff
@@ -1210,6 +1210,40 @@ class DeltaTableWritesSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBa
     }
   }
 
+  test("update table properties on a column mapping enabled table") {
```

Comment:
this wasn't super intuitive to me at first to understand the issue, can you confirm that I understand correctly?

- before, we were passing only the `newProperties` as the new config
- this means we would identify the `newMappingMode=none` and then fail?

Reply:
thats correct.